### PR TITLE
added spanning-tree portfast trunk to ShowRunInterface parser

### PR DESCRIPTION
**Pull request by smitsialis**
## Description
Added a parser to ShowRunInterface for the following command:
 * spanning-tree portfast trunk
Also found variable in the schema that were duplicates.  Removed the duplicates.

## Motivation and Context
This was missing and is needed.

## Impact (If any)
no impact

## Screenshots:
image

## Checklist:

- [x] I have updated the changelog.
- [ ] I have updated the documentation (If applicable).
- [x] I have added tests to cover my changes (If applicable).
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_run.py [hunk 1]
```diff
@@ -277,8 +277,6 @@ class ShowRunInterfaceSchema(MetaParser):
                 Optional('snmp_trap_link_status'): bool,
                 Optional('snmp_trap_mac_notification_change_added'): bool,
                 Optional('snmp_trap_mac_notification_change_removed'): bool,
-                Optional('spanning_tree_bpduguard'): str,
```
Comment (lsheikal) [inline src/genie/libs/parser/iosxe/show_run.py:280]: Why these keys have been removed from parser?

File: src/genie/libs/parser/iosxe/show_run.py [hunk 2]
```diff
@@ -277,8 +277,6 @@ class ShowRunInterfaceSchema(MetaParser):
...
-                Optional('spanning_tree_portfast'): bool,
```
Comment (lsheikal) [inline src/genie/libs/parser/iosxe/show_run.py:281]: same. why its been removed?

File: changelog/undistributed/changelog_iosxe_ShowRunInterface_20210923201701.rst [hunk 3]
```diff
@@ -0,0 +1,16 @@
+--------------------------------------------------------------------------------
```
Comment (lsheikal) [inline changelog/undistributed/changelog_iosxe_ShowRunInterface_20210923201701.rst:1]:
```suggestion
--------------------------------------------------------------------------------
                                  Fix
--------------------------------------------------------------------------------
* IOSXE
    * Modified ShowRunInterface:
        * Added new key spanning-tree portfast trunk
        * Removed duplicate keys spanning_tree_bpduguard and spanning_tree_portfast
```

**Review by lsheikal — CHANGES_REQUESTED** [review]

File: src/genie/libs/parser/iosxe/show_run.py (hunk 1 above)
Comment (danielgraziano) [reply src/genie/libs/parser/iosxe/show_run.py:280]: they are duplicates

File: src/genie/libs/parser/iosxe/show_run.py (hunk 2 above)
Comment (danielgraziano) [reply src/genie/libs/parser/iosxe/show_run.py:281]: they are duplicates

File: changelog/undistributed/changelog_iosxe_ShowRunInterface_20210923201701.rst [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+            * spanning-tree portfast trunk
+
+--------------------------------------------------------------------------------
+                            Fix
+--------------------------------------------------------------------------------
+* IOSXE
+    * Modified ShowRunInterface:
+        * Removed duplicate schema variables
+                * Optional('spanning_tree_bpduguard'): str,
+                * Optional('spanning_tree_portfast'): bool,
```
Comment (danielgraziano) [inline changelog/undistributed/changelog_iosxe_ShowRunInterface_20210923201701.rst:16]:
```suggestion
--------------------------------------------------------------------------------
                            New
--------------------------------------------------------------------------------
* IOSXE
    * Modified ShowRunInterface:
        * Added parsing support (schema and parsers) for following output
            * spanning-tree portfast trunk

--------------------------------------------------------------------------------
                            Fix
--------------------------------------------------------------------------------
* IOSXE
    * Modified ShowRunInterface:
        * Removed duplicate schema variables
                * Optional('spanning_tree_bpduguard'): str,
                * Optional('spanning_tree_portfast'): bool,
        * Added the following schema variable
                * Optional('spanning_tree_portfast_trunk'): bool,
```

**Review by danielgraziano — CHANGES_REQUESTED** [review]
Please update the changelog

**Comment by smitsialis** [comment]
@danielgraziano  I had made the changes and re-submitted them before I saw there was a button to just approve.  I've re-requested a review.  Not sure where to go from here.  Let me know what I need to do.  Thanks,

**Comment by danielgraziano** [comment]
@smitsialis Unfortunately you'll have to follow the below steps. We had to make some changes to the repository that are incompatible with any existing clones or forks.

First you'll have to save your parsers that you wrote and want to contribute.

Second, you'll need to delete your copy of the master (smitsialis:master) and then re-pull the master branch from the remote (CiscoTestAutomation:master). This step is required because the history had to be re-written :(.

Third, copy your saved parsers/tests back into your copy of master (smitsialis:master) and then create a pull request

**Comment by danielgraziano** [comment]
Alternatively just save your changes, delete the fork, re-fork, copy your changes back in

**Pull request closed without merging**